LOH/1.0.0/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:22.04
+LABEL maintainer="Saksham Phul (phuls@chop.edu)"
+
+WORKDIR home/
+
+RUN apt update && apt upgrade -y && \
+        apt install -y wget && \
+        apt install -y python3.10 && \
+        wget https://github.com/samtools/bcftools/releases/download/1.16/bcftools-1.16.tar.bz2 && \
+        apt install bzip2 && \
+        tar -axf bcftools*.tar.bz2 && rm bcftools*.tar.bz2 && bash && \
+        cd bcftools* && apt install -y gcc make liblzma-dev zlib1g-dev libbz2-dev && \
```

Comment:
does this cd syntax actually work? `cd bcftools*` is not a well-defined path i don't think